test/contributionreward.js

```diff
@@ -72,10 +72,10 @@ contract('ContributionReward', function(accounts) {
      assert.equal(await testSetup.contributionReward.isRegistered(testSetup.org.avatar.address),true);
     });
 
-    it("submitContribution log", async function() {
+    it("proposeContributionReward log", async function() {
       var testSetup = await setup(accounts,0,0);
       // await testSetup.contributionReward.registerOrganization(testSetup.org.avatar.address);
```

Comment:
Though this PR did not update that ..
I would advice to remove this commented code.